runner/src/main.rs

```diff
@@ -320,7 +320,7 @@ fn build_server(opt: &BuildServer) -> Step {
                             // If building in coverage mode, use the default target from the host, and build
                             // in debug mode.
                             ServerVariant::Coverage => vec!["--features=oak_unsafe,oak_introspection_client".to_string()],
-                            ServerVariant::Experimental => vec!["--features=oak_attestation,oak_unsafe,oak_introspection_client".to_string(),
+                            ServerVariant::Experimental => vec!["--features=oak_attestation".to_string(),
```

Comment:
`abitest` needs `oak_unsafe`, because it uses introspection functionality. So, if you don't want to have `oak_unsafe` with `experimental`, it cannot be the default server variant for running examples in the CI builds.

Why is the default set to `experimental`?